tests/queries/0_stateless/01691_date_ubsan.sql

```diff
@@ -0,0 +1,5 @@
+-- Original from fuzzer
+SELECT toString(toStartOfHour(toDateTime(1024, toDateTime(toDateTime(toDateTime(toDateTime(NULL)), toDateTime(toDateTime(1024, toString(toDateTime(1048576, NULL)), NULL))), toDateTime(NULL)), toString(toDateTime(-2, 7)), NULL), toDateTime(3, NULL), toDateTime(toDateTime(toDateTime(toDateTime(toDateTime(NULL))), 255), 1024, toString(toDateTime(toString(toDateTime(65535, toString(toDateTime(toString(toStartOfHour(toDateTime(0.)), toDateTime(257, NULL)))), NULL), toStartOfHour(toDateTime(1.)), toDateTime(256, NULL)), -9223372036854775808)), NULL)), 'Europe/London') FORMAT Null;
+
+-- Minimized example
+SELECT toDateTime(-2, 7);
```

Comment:
Why we got `02:00:14`? Maybe we should forbid negative arguments at all?

Reply:
>Maybe we should forbid negative arguments at all?

Sounds good, will rebase the patch